addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -350,3 +350,35 @@ describe("Link formatting in the popover", () => {
         expect(linkPreviewEl).toHaveText("newtest.com");
     });
 });
+
+describe("shortcut", () => {
+    test("create a link with shortcut", async () => {
+        const { el } = await setupEditor(`<p>[]</p>`);
+
+        press(["ctrl", "k"]);
+        await animationFrame();
+        click(".o_command_name:first");
+        await waitFor(".o-we-linkpopover");
+
+        queryOne(".o_we_href_input_link").focus();
+        for (const char of "test.com") {
+            press(char);
+        }
```

Comment:
Can't we just use `contains(...).edit(...)` to edit the input ?

Reply:
I will have a try :smile_cat: